modules/persistent-storage-local-tolerations.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// storage/persistent-storage/persistent-storage-local.adoc
+
+[id="local-tolerations_{context}"]
+= Using tolerations with Local Storage Operator Pods
+
+You can control which nodes the Local Storage Operator Pod runs on and prevent
+other workloads from using those nodes by using tolerations on the Pods.
+
+You apply tolerations to the Local Storage Operator Pod through the LocalVolume Custom Resource (CR)
+and apply taints to a node through the node specification. A taint on a node is a `key:value pair` that
+instructs the node to repel all Pods that do not tolerate the taint. Using a specific `key:value` pair
+that is not on other Pods ensures that only the Local Storage Operator Pod can run on that node.
+
+.Prerequisites
+
+* The Local Storage Operator is installed.
+
+* Local disks are attached to the {product title} nodes.
+
+.Procedure
+To allow the LocalVolume CR to select tainted nodes for use:
+
+. Modify the YAML file that defines the Pod and add the `LocalVolume` spec, as shown in the following example:
++
+[source, yaml]
+----
+  apiVersion: "local.storage.openshift.io/v1"
+  kind: "LocalVolume"
+  metadata:
+    name: "local-disks"
+    namespace: "local-storage"
+  spec:
+    tolerations:
+      - key: localstorage <1>
+        operator: Equal <2>
+        value: "local"
```

Comment:
Since you've added annotations for the first two, you should probably add it for the third as well.

Note that if the operator is `Exists`, then it simply checks that the key exists and ignores the value. If the operator is `Equal`, then the key **and** value must match. It might be worth a mention here?

Reply:
Agree. Added the third annotation and a note to #2.